### ISSUE TITLE
 Failed to execute goal org.owasp:dependency-check-maven:5.1.1:check 

### ISSUE THREAD

**Issue by Jezwin** [Adobe-Consulting-Services/acs-aem-commons#2117]
Getting this error on -PautoInstallPackage
Failed to execute goal org.owasp:dependency-check-maven:5.1.1:check (check-dependencies-for-vulnerabilities) on project acs-aem-commons-bundle: One or more exceptions occurred during dependency-check analysis: One or more exceptions occurred during analysis:

**Comment by eberggren343** [comment]
I receive the same error when trying to `mvn clean package`. Downloaded and unzipped **4.3.4** source code and trying to build.

[ERROR] Failed to execute goal org.owasp:dependency-check-maven:5.1.1:check (check-dependencies-for-vulnerabilities) on project acs-aem-commons: Fatal exception(s) analyzing ACS AEM Commons - Reactor Project: One or more exceptions occurred during analysis:
[ERROR]         Failed to initialize the RetireJS repo
[ERROR]         No documents exist

**Comment by joerghoh** [comment]
```
unable to find valid certification path to requested target
```
@eberggren343 your exception looks like that you have issues with SSL certificates. Are you behind a corporate firewall (but also some desktop firewalls are fiddling with network traffic and certificates).

**Comment by joerghoh** [comment]
works here with 
```
Maven home: /usr/local/Cellar/maven/3.5.3/libexec
Java version: 1.8.0_172, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre
Default locale: en_DE, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```

and also with openjdk 11 on Linux.

**Comment by kwin** [comment]
This should be solved with #2166

**Comment by joerghoh** [comment]
The dependency check is not executed locally anymore (#2166 is closed already).
